src/main/java/Duke.java

```diff
@@ -1,10 +1,255 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String FILE_PATH = "./data/duke.txt";
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        List<Task> taskList = loadTasksFromFile();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Duke101\n"
+                + "What can I do for you?");
+
+        while (true) {
+            String userInput = scanner.nextLine();
+
+            if (userInput.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            try{
+                processCommand(userInput, taskList);
```

Comment:
I like how you extracted the command processing to a separate function to make the `main` function more readable!